university-agent/src/main/java/nl/quintor/studybits/service/StudentService.java

```diff
@@ -75,21 +78,38 @@ public void setStudentDid(String studentId, String studentDid) {
 
     @Transactional
     public Student createStudent() {
-        Student student = new Student();
-        student.setStudentId(UUID.randomUUID().toString());
+        String randomId = UUID.randomUUID().toString();
+        if(!studentExists(randomId)) {
```

Comment:
The odds of this happening are so extremely tiny that I would be very okay with leaving this check out.